deps.edn

```diff
@@ -48,7 +48,8 @@
    :extra-deps  {org.clojure/tools.namespace {:mvn/version "1.5.0"}
                  clj-http/clj-http           {:mvn/version "3.13.0"}
                  criterium/criterium         {:mvn/version "0.4.6"}
-                 integrant/repl              {:mvn/version "0.3.3"}}}
+                 integrant/repl              {:mvn/version "0.3.3"}}
+   :jvm-opts    ["-Djdk.attach.allowAttachSelf"]}
```

Comment:
What is this for?

Reply:
This allows me to cancel a running process from the repl in emacs - I accidentally hung the whole jvm while adding drop support and this was how I was able to debug it. I didn't intend to add it to git, but I don't think it does any harm and is useful in general for development. I'm fine removing it, but if you're fine then I wouldn't mind keeping it around.